x-pack/plugins/apm/server/saved_objects/apm_service_groups.ts

```diff
@@ -7,8 +7,37 @@
 
 import { SavedObjectsType } from '@kbn/core/server';
 import { i18n } from '@kbn/i18n';
+import type { SavedObjectMigrationFn } from '@kbn/core/server';
 import { APM_SERVICE_GROUP_SAVED_OBJECT_TYPE } from '../../common/service_groups';
 
+interface ApmServiceGroupsPre850 {
+  groupName: string;
+  kuery: string;
+  description: string;
+  serviceNames: string[];
+  color: string;
+}
+
+interface ApmServiceGroups850 {
```

Comment:
I don't think you have to add a version to this one since you've already specified the version in the other one.

```suggestion
interface ApmServiceGroups {
```